_locales/en/messages.json

```
@@ -390,6 +390,8 @@
     "backup_file_info": {
         "message": "Backup your data to a file."
     },
+    "password_policy_default_hint": {
+        "message": "The password doesn't meet the security requirements. Contact with your administrator."
```

```suggestion
        "message": "Your password does not meet your organization's security requirements. Contact your administrator for more information."
    },
```

This is why the tests are failing. The json is invalid, causing the extension to not load. Therefore, the test runner is unable to communicate with view/test.html